docs/monitoring/metrics.md

```diff
@@ -658,6 +658,7 @@ Parameters:
 - `db` - the InfluxDB database to store metrics
 - `username` - (optional) InfluxDB username used for authentication
 - `password` - (optional) InfluxDB username's password used for authentication
+- `rp` - (optional) InfluxDB retention policy, defaults to retention policy defined on the server
```

Comment:
Updates in `metrics.md` should also be updated in `metrics-zh.md`. Moreover, `metrics.hmtl` should also be updated, please refer to `flink-docs/README.md` to generate docs.

Reply:
You are right @Myasuka - the retention policy needs to created when creating the database.  Also, let me now what's the process the other docs you've mentioned.

Reply:
Please refer to [generate config doc](https://github.com/apache/flink/blob/master/flink-docs/README.md#configuration-documentation) to know the process. And you could search the `metrics-zh.md` to make they are the same as English version.

Reply:
I've updated the metrics.zh.md - thanks for the pointer.